soc/projects/numerics.md

```diff
@@ -82,6 +93,8 @@ and boundary cases.
 
 **Expected Results**: New and faster methods for evaluating properties of special functions.
 
+**Mentors:** [Alex Arslan](https://github.com/ararslan)
```

Comment:
Thanks for the link! Maybe next year I'll volunteer to mentor for a project I'm more intimately familiar with. 🙂